sys-apps/shadow/shadow-4.8-r2.ebuild

```diff
@@ -148,8 +148,7 @@ src_install() {
 			newpamd "${FILESDIR}"/pam.d-include/passwd ${x}
 		done
 
-		for x in groupmems chage chpasswd chgpasswd newusers \
-				 user{add,del,mod} group{add,del,mod} ; do
+		for x in groupmems ; do
```

Comment:
you probably do not want to have a loop for a single iteration.

Reply:
Removed. I wanted to keep the notion that this template can be installed for multiple tools but it's certainly not necessary.